### PR TITLE
[#465634]Fix members in Process designer

### PR DESCRIPTION
**Pull request by pablo3443**

### CODE REVIEW

File: tests/utils/memberData.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export const data = {
+    available: [
+        { id: 'account.3', type: 'users' }, 
+        { id: 'account.4', type: 'users' }
+    ],
+    selected: [
+        { id: 'account.7', type: 'users' },
+        { id: 'account.8', type: 'users' },
+        { id: 'account.9', type: 'groups' }
```
Comment (PavelKoroteev) [inline tests/utils/memberData.js]: account.9 ---> group.9

File: tests/specs/form/MemberSplitEditor.spec.js [hunk 2]
```diff
@@ -59,6 +60,63 @@ describe('Editors', () => {
...
+        it('should work with custom display text', done => {
+            const model = new Backbone.Model({
+                selected: ['dog.1', 'dog.2', 'dog.4']
+            });
+            const view = new core.form.editors.MembersSplitEditor({
+                model,
+                key: 'selected',
+                autocommit: true,
+                showMode: 'button',
+                getDisplayText: dogs => 'Here I have '.concat(dogs.length).concat(' dogs')
```
Comment (PavelKoroteev) [inline tests/specs/form/MemberSplitEditor.spec.js]: template string ?

File: src/form/editors/impl/membersSplit/controller/MembersSplitController.ts [hunk 3]
```diff
@@ -73,41 +76,53 @@ export default Marionette.MnObject.extend({
...
             this.__setLoading(false);
         }
     },
 
-    __getSettings(filterState) {
-        const selectedModels = this.model.initialized ? this.model.get('selected').parentCollection.map(item => item.id) : this.options.selected;
+    async __updateGeneralItems() {
+        this.members = {};
+        this.__setLoading(true, { both: false });
+        const users = await this.options.users;
```
Comment (PavelKoroteev) [inline src/form/editors/impl/membersSplit/controller/MembersSplitController.ts]: try catch

**Review by PavelKoroteev — APPROVED** [review]

File: tests/utils/memberData.js (hunk 1 above)
Comment (pablo3443) [reply tests/utils/memberData.js]: Done

File: tests/specs/form/MemberSplitEditor.spec.js (hunk 2 above)
Comment (pablo3443) [reply tests/specs/form/MemberSplitEditor.spec.js]: Done

File: src/form/editors/impl/membersSplit/controller/MembersSplitController.ts (hunk 3 above)
Comment (pablo3443) [reply src/form/editors/impl/membersSplit/controller/MembersSplitController.ts]: Done

**Pull request merged**